formazione/templates/pdf_firme_lezione.html

```diff
@@ -0,0 +1,107 @@
+{% load static %}
```

Comment:
Ti invito a guardare tutti gli altri template di PDF (tutti i file `pdf_*.html`). I modelli dei PDF estendono `pdf_standard.html` che ha gia' la maggior parte di questo codice; non devi scrivere tutta l'intestazione e il pie' di pagina per ogni PDF, e puoi anche rimuovere il CSS in eccesso.

Reply:
avevo iniziato così ma pdf_standard definisce un footer fuori dal corpo, nell'elenco firme il footer è per pagina (o per qualche magia di dompdf un p con id footer_pdf lo fa su ogni pagina?)

Reply:
Se hai un footer per pagina, credo tu possa semplicemente inserire il tuo footer come parte del corpo, in ogni pagina. Se hai necessita' di nascondere o modificare un blocco definito da un modello genitore, puoi sovrascriverlo.

Reply:
se sovrascrivo il blocco html di pdf_standard praticamente estendo pdf_vuoto che è
{% block html %}
<!-- Gaia PDF -->
{% endblock %}

il footer non è sovrascrivibile, e in ogni caso load static tocca metterlo comunque
Come ti dicevo avevo già guardato i template ma non mi sembrava fattibile

Reply:
Quello che intendo dire con
> Se hai un footer per pagina, credo tu possa semplicemente inserire il tuo footer come parte del corpo, in ogni pagina.
E' che non c'e' bisogno di sovrascrivere niente, semplicemente scrivi tutto come parte del corpo del PDF (`{% block corpo %}`, per intenderci).

Reply:
si ma in pdf_standard il footer è fuori il blocco corpo a me ne serve uno per pagina

Reply:
@acarmisciano, stiamo avendo delle difficolta' di comunicazione. Fammi essere piu' chiaro: cosa intendi per footer? Parli di questo blocco?

```
<div class="footer">
  Documento stampabile generato digitalmente da <strong>Gaia</strong>. Autenticato: {{ timestamp }}
</div>
```

Questo non e' importante, sicuramente non e' necessario in ogni pagina!

Reply:
si è quello ma se metto il mio codice in `corpo` e lascio il footer il risultato è tutte le pagine senza footer tranne l'ultima col footer, se per voi è ok

Reply:
Come in tutti gli altri PDF su Gaia, "*Documento stampabile generato digitalmente da Gaia*" va apposto solo sull'ultima pagina. Quindi si, va bene.

Reply:
ok